docs: Document the limitation between uib and app directory (#2869)
bonitasoft/bonita-doc: modules/runtime/pages/application-directory.adoc

. In the menu, click on "Applications"
. On the "Bonita Application Directory" row, click on the "..." _View application details_ icon

> WARNING:

Reviewer: L'affichage du warning est cassé sur la preview : 
<img width="889" alt="image" src="https://github.com/user-attachments/assets/f48bf22d-38ea-4c25-a910-04e407d9e039">

> WARNING:

Author: ```suggestion
[WARNING]
```

> --

Author: ```suggestion
==
```

> On development phase, if you try to access to a xref:applications:ui-builder/version-control-with-git/version-control-with-git.adoc[UI-Builder application connected to Git repository] from the Application Directory, the redirection will fail.

Reviewer 2: ```suggestion
During the development phase, if you try to access a xref:applications:ui-builder/version-control-with-git/version-control-with-git.adoc[UI-Builder application connected to a Git repository] from the Application Directory, the redirection will fail.
```

> To access to the UI-Builder application, you need to access it directly from the Ui-Builder URL.

Reviewer: ```suggestion
To access the UI Builder application, you need to access it directly from the URL (e.g. `http://localhost:8090/app/your-application-name/…`).
```

Author: ```suggestion
To access the UI Builder application, you need to access it directly from the URL (e.g `http://localhost:8090/`)
```

> --

Author: ```suggestion
==
```

> To access the UI Builder application, you need to access it directly from the URL (e.g `http://localhost:8090/`)

Reviewer 2: ```suggestion
To access the UI Builder application, you need to access it directly from the U Builder workspace (e.g `http://localhost:8090/`)
```
